server/test/services/applications/ProgramAdminApplicationServiceTest.java

```diff
@@ -361,6 +570,76 @@ public void setStatus_tiApplicant_sendsEmail() throws Exception {
                 STATUS_WITH_ONLY_ENGLISH_EMAIL.localizedEmailBodyText().get().getDefault()));
   }
 
+  @Test
+  public void setStatuses_tiApplicant_sendsEmail_nonDefaultLocale() throws Exception {
+    String userEmail = "user@email.com";
+    String tiEmail = "ti-ko@email.com";
+    EmailSendClient emailSendClient = Mockito.mock(EmailSendClient.class);
+    MessagesApi messagesApi = instanceOf(MessagesApi.class);
```

Comment:
Suggestion (non-blocking): The code to create the service is repeated several times in different tests. Would it make sense to make a helper method?

Reply:
sure!

Reply:
unfortunately, emailSendClient is needed for the verify() step of the steps.

Reply:
I did push the code creating the service with mocked object to helper method